src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.dialog.html

```diff
@@ -1,35 +1,60 @@
-        <umb-control-group label="{{selectContentTypeLabel}}" ng-switch-when="selectDocType">
-            <ul class="umb-actions umb-actions-child">
-                <li data-element="action-create-{{docType.alias}}" ng-repeat="docType in docTypes | orderBy:'name':false">
-                    <a ng-click="selectDocType(docType.alias)">
-                        <i class="large {{docType.icon}}"></i>
-                        <span class="menu-label">
-                            {{docType.name}}
-                            <small>
-                                {{docType.description}}
-                            </small>
-                        </span>
-                    </a>
-                </li>
-            </ul>
-        </umb-control-group>
-
-        <div ng-switch-when="edit">
-
-            <fieldset ng-repeat="tab in model.node.tabs">
-                <legend style="font-size: 18px;">{{tab.label || tab.alias}}</legend>
-
-                <umb-property property="property"
-                              ng-repeat="property in tab.properties">
-                    <umb-editor model="property"></umb-editor>
-                </umb-property>
-
-            </fieldset>
-
-        </div>
-    </div>
+<div class="dtge-dialog" ng-controller="Our.Umbraco.DocTypeGridEditor.Dialogs.DocTypeGridEditorDialog as vm">
+        
+    <umb-editor-view>
+        <umb-editor-header name="model.title"
+                           name-locked="true"
+                           hide-alias="true"
+                           hide-icon="true"
+                           hide-description="true">
+        </umb-editor-header>
+        <umb-editor-container ng-if="!vm.loading && dialogMode == 'edit'">
+            <ng-form val-form-manager>
+                <umb-tabbed-content content="vm.content"></umb-tabbed-content>
+            </ng-form>
+        </umb-editor-container>
+        <umb-editor-container ng-if="!vm.loading && dialogMode == 'selectDocType'">
+            <umb-box>
+                <umb-box-content>
+                    <div class="form-search" ng-hide="model.filter === false" style="margin-bottom: 15px;">
+                        <i class="icon-search"></i>
+                        <input type="text"
+                               ng-model="searchTerm"
+                               class="umb-search-field search-query input-block-level -full-width-input"
+                               localize="placeholder"
+                               placeholder="@placeholders_filter"
+                               umb-auto-focus
+                               no-dirty-check />
+                    </div>
 
+                    <ul class="umb-card-grid -three-in-row">
+                        <li ng-repeat="docType in docTypes | orderBy:'name' | filter:searchTerm"
+                            ng-click="selectDocType(docType.alias)">
+                            <a class="umb-card-grid-item" href="" title="{{ docType.name }}">
```

Comment:
@skttl can we update `title` to `ng-attr-title` instead, which seems to be supported since Angular v1.1.4?
https://stackoverflow.com/a/18346925/1693918

It seems it also should be fixed in core
https://github.com/umbraco/Umbraco-CMS/blob/1815112ae7a3433ca829bd5537f4417fea883b38/src/Umbraco.Web.UI.Client/src/views/common/infiniteeditors/datatypepicker/datatypepicker.html#L50